src/components/forms/PSESponsorshipsForm.tsx

```diff
@@ -0,0 +1,1255 @@
+            )}
+          </FormControl>
+
+          <FormControl id='referrals-control' mb={8}>
+            <FormLabel htmlFor='referrals' mb={1}>
+              <PageText fontSize='input'>Who referred you?</PageText>
+            </FormLabel>
+
+            <PageText as='small' fontSize='helpText' color='brand.helpText'>
+              Please write the name of the person who shared this form with you.
+            </PageText>
+
+            <Textarea
+              id='referrals'
+              bg='white'
+              borderRadius={0}
+              borderColor='brand.border'
+              _placeholder={{ fontSize: 'input' }}
+              color='brand.paragraph'
+              fontSize='input'
+              h='150px'
+              mt={3}
+              {...register('referrals', {
+                maxLength: 32768
+              })}
+            />
+
+            {errors?.referrals?.type === 'maxLength' && (
+              <Box mt={1}>
+                <PageText as='small' fontSize='helpText' color='red.500'>
+                  Referrals info cannot exceed 32768 characters.
+                </PageText>
+              </Box>
+            )}
+          </FormControl>
+
+          <FormControl id='pse-rationale-control' mb={8}>
+            <FormLabel htmlFor='pseRationale' mb={1}>
+              <PageText fontSize='input'>
+                What do you hope PSE&apos;s support will add to this event?
+              </PageText>
+            </FormLabel>
+
+            <PageText as='small' fontSize='helpText' color='brand.helpText'>
+              Are there specific topics or values that are especially relevant to PSE? Why PSE
+              rather than EF generally?
+            </PageText>
+
+            <Textarea
+              id='pseRationale'
+              bg='white'
+              borderRadius={0}
+              borderColor='brand.border'
+              _placeholder={{ fontSize: 'input' }}
+              color='brand.paragraph'
+              fontSize='input'
+              h='150px'
+              mt={3}
+              {...register('pseRationale', {
+                maxLength: 32768
+              })}
+            />
+
+            {errors?.pseRationale?.type === 'maxLength' && (
+              <Box mt={1}>
+                <PageText as='small' fontSize='helpText' color='red.500'>
+                  The reasons cannot exceed 32768 characters.
+                </PageText>
+              </Box>
+            )}
+          </FormControl>
+
+          <Flex direction={{ base: 'column', lg: 'row' }}>
+            <Controller
+              name='eventType'
+              control={control}
+              rules={{
+                required: true,
+                validate: selected => selected.value !== ''
+              }}
+              defaultValue={{ value: '', label: '' }}
+              render={({ field: { onChange }, fieldState: { error } }) => (
+                <FormControl id='event-type-control' mb={8} mr={{ md: 12 }}>
```

Comment:
```suggestion
                <FormControl id='event-type-control' mb={8} mr={{ md: 12 }} isRequired>
```

Reply:
`Event type` is not required for this form

Reply:
If you look at the wrapper `Controller`, it has a `required: true` rule, making this field as a required one. This would just add the `*` to the UI.

If the field is not required then we should remove the rules in line L850 to avoid having misinterpretations with the `isValid` flag.